### PR TITLE
test: :white_check_mark: Add e2e test on user module

### PR DESCRIPTION
**Pull request by javillegasna**
## Jira Link

[PSB-4](https://trello.com/c/XlpDAQdx/14-psb-4-implement-e2e-test-on-user-module)

## Types of changes

- [ ] Hotfix (Bug detected)
- [ ] Feature (Issue from an Active Sprint)
- [x] Test ( Testing Issue from an Active or previous Sprint)
- [ ] Refactor (Issue from a Previous Sprint)
- [ ] Documentation (Add or update documentation)
## Checklist:

- [x] Unit Test.
- [x] E2E Test
- [x] Coverage.

## Solution
A recover function has been added at user service and resolver,  also e2e test of the user module has been added

### CODE REVIEW

**Comment by javillegasna** [comment]
[PSB-4-Implement e2e test on user module](https://trello.com/c/XlpDAQdx/14-psb-4-implement-e2e-test-on-user-module)

File: test/users/users.e2e-spec.ts [hunk 1]
```diff
@@ -0,0 +1,202 @@
+import { Test, TestingModule } from '@nestjs/testing';
```
Comment (SebasVA1234) [inline test/users/users.e2e-spec.ts]: Please, put in order the imports using the covetion.

File: test/users/users.e2e-spec.ts [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+          users {
+            name,
+            lastName
+          }
+        }`,
+      })
+      .expect(200);
+
+    const { users } = response.body.data;
+
```
Comment (SebasVA1234) [inline test/users/users.e2e-spec.ts:91]: Create a new spect that verify the length of the array must be one.

File: test/users/users.e2e-spec.ts [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+        variables: { recoverUserId: userId },
+      })
+      .expect(200);
+
+    const { recoverUser } = response.body.data;
+
+    expect(recoverUser).toEqual({ id: userId });
+  });
+
+  it('/ User Resolver (mutation recoverUser)', async () => {
```
Comment (SebasVA1234) [inline test/users/users.e2e-spec.ts]: Change the description of the test, becouse this is permanent remove.

**Review by SebasVA1234 — APPROVED** [review]

**Pull request merged**